bindings/java/src/main/java/org/ucx/jucx/ucs/UcsConstants.java

```diff
@@ -5,19 +5,23 @@
 
 package org.ucx.jucx.ucs;
 
+import org.ucx.jucx.NativeLibs;
+
 public class UcsConstants {
+    static {
+        NativeLibs.load();
+        loadConstants();
+    }
 
     /**
-     * Thread sharing mode
-     *
-     * <p>Specifies thread sharing mode of an object.
+     * Only the master thread can access (i.e. the thread that initialized the context;
+     * multiple threads may exist and never access)
      */
-    public enum UcsThreadMode {
-        UCS_THREAD_MODE_SINGLE,     /**< Only the master thread can access
-                                         (i.e. the thread that initialized the context;
-                                         multiple threads may exist and never access) */
-        UCS_THREAD_MODE_SERIALIZED, /**< Multiple threads can access, but only one at a time */
-        UCS_THREAD_MODE_MULTI,      /**< Multiple threads can access concurrently */
-        UCS_THREAD_MODE_LAST
-    }
+    public static int UCS_THREAD_MODE_SINGLE;
```

Comment:
Lets incapsulate into `ThreadMode` as before

Reply:
`ThreadMode` was enum. In java enum is static enumeration defined at compile time:
https://stackoverflow.com/questions/478403/can-i-add-and-remove-elements-of-enumeration-at-runtime-in-java

You can't set new value at runtime for enum.

Reply:
you don't need add or remove elements, you have to initialize them.
if understand correctly it's described here: https://stackoverflow.com/questions/8811815/is-it-possible-to-assign-numeric-value-to-an-enum-in-java/8811869#8811869

can we read native values in constructor of enum?

if it's impossible lets wrap the constants into static class instance or singleton

Reply:
Let's return to the original version with enum and not set from jni.

Reply:
Ok but without serialized mode